### ISSUE TITLE
Fix for sklearn on GraalVM

### ISSUE THREAD

**Issue by ashesfall** [oracle/graalpython#263]
I am interested in contributing a solution to support sklearn on GraalVM. I realize that there may be substantial barriers to this, but after compiling it from source against the numpy version that is already already available on GraalVM, it seems it may be possible.

I don't see any information about how to do this though. Installation with pip on graalvm fails, but there seems to be no documentation about how to build a new package that fixes the issues raised by such a failure.

**Comment by timfel** [comment]
Hi, we've been running sklearn with just a small set of patches internally for some years, but never got around to publishing the patch. Your question comes at a convenient time, since we put publication of that on our current issue list just a couple of weeks ago, so it'll be published soon.

**Comment by ashesfall** [comment]
Is there a [public] place where this is tracked? We have a batch process combining Java and Python (currently with communication over HTTP instead of together as one application) and its an ideal candidate to potentially demonstrate a more-than-hypothetical Graal Python use case. I'm anxious to know if the timeframe is in months or in days.

**Comment by timfel** [comment]
Be aware, though, that our performance on sklearn is not great, in our benchmarks you can expect a long warmup time and a peak performance well below CPython

**Comment by cosminbasca** [comment]
hi @Jnnsm, thank you for your interest. The fix should be in in the next few weeks.

**Comment by timfel** [comment]
scikit-learn can now be installed and used via pip as of the 23.0 release

**Comment by Jnnsm** [comment]
Hi, so I was trying to run my tests again and found some problems while installing scikit-learn

```
Build type: native build
            Project name: SciPy
            Project version: 1.10.1
            C compiler for the host machine: cc (clang 14.0.0 "Apple clang version 14.0.0 (clang-1400.0.29.202)")
            C linker for the host machine: cc ld64 820.1
            C++ compiler for the host machine: c++ (clang 14.0.0 "Apple clang version 14.0.0 (clang-1400.0.29.202)")
            C++ linker for the host machine: c++ ld64 820.1
      
            ../../meson.build:1:0: ERROR: Cython compiler 'cython' cannot compile programs
```

I tried running with the "--install-option="--no-cython-compile"" but still no luck. Any tips?
